.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
 -   repo: local
     hooks: 
     - id: format_checkers
-      language: python
+      language: system
       name: format_checkers
       entry: python cve_bin_tool/format_checkers.py
       files: "^cve_bin_tool/checkers/__init__.py"
```

Comment:
Note: I manually removed it (temporarily) to generate the updated markdown files.